zeekpkg/uservar.py

```diff
@@ -137,7 +138,6 @@ def resolve(
 
         desc = " (" + self._desc + ")" if self._desc else ""
         print(f'"{name}" requires a "{self._name}" value{desc}: ')
-        assert val
         self._val = _rlinput(self._name + ": ", val)
```

Comment:
I'd probably do something like this instead and leave the typing of `_rlinput()` alone. A function having a default value of `""`, but then also accepting a `None` even though `""` and `None` are treated the same tickles "this is strange".

```suggestion
        self._val = _rlinput(self._name + ": ", prefill=val if val else "")
```

But don't think that matters enough really.

Reply:
Good idea.

TBH, I was slightly shocked how little test coverage we had since the reproducer was so basic. Let's hope we can shake out any remaining issues before 8.1.